src/twisted/trial/_dist/worker.py

```diff
@@ -185,7 +185,7 @@ def addExpectedFailure(
         """
         Add an expected failure to the reporter.
         """
-        _todo = Todo(todo)
+        _todo = Todo("<unknown>" if todo is None else todo)
```

Comment:
I guess that this change was done to make it easier to remove `Todo.__init__` while defining `reason: str` and not `Optional[str]`

I am happy to always have a string as the reason.

Reply:
Yep.  I also took a hint from the existing docstring for `Todo` which claimed `reason` was a string and said nothing about None.